jax/_src/xla_bridge.py

```diff
@@ -333,8 +334,10 @@ def make_gpu_client(
   )
   if platform_name == "cuda":
     _check_cuda_versions()
-    devices_to_check = allowed_devices if allowed_devices else range(cuda_versions.cuda_device_count())
-    _check_cuda_compute_capability(devices_to_check)
+    # TODO(micky774) remove this check when minimum jaxlib is v0.4.26
+    if version.__version_info__ > (0, 4, 26):
+      devices_to_check = allowed_devices if allowed_devices else range(cuda_versions.cuda_device_count())
```

Comment:
Can you keep this line within 80 characters please? You can set that as the restriction in your IDE.